### PR TITLE
add support for top-level property refInFor

### PR DESCRIPTION
**Pull request by baiyaaaaa**

### CODE REVIEW

**Comment by baiyaaaaa** [comment]
@yyx990803 hi, please give a suggestion^_^

File: test/test.js [hunk 1]
```diff
@@ -34,13 +34,15 @@ describe('babel-plugin-transform-vue-jsx', () => {
...
         ref="ref"
+        refInFor
         slot="slot">
       </div>
     ))
     expect(vnode.data.class).to.equal('foo')
     expect(vnode.data.style).to.equal('bar')
     expect(vnode.data.key).to.equal('key')
     expect(vnode.data.ref).to.equal('ref')
+    expect(vnode.data.refInFor).to.true
```
Comment (yyx990803) [inline test/test.js]: `.to.be.true`?

**Comment by baiyaaaaa** [comment]
done

**Comment by yyx990803** [comment]
out in 3.3.0

**Pull request merged**